### DISCUSSION TITLE
Xbox compatible controller with ev3

### DISCUSSION THREAD

**Discussion by noravanq** [pybricks/support#1639] [Q&A]
Are Xbox compatible controllers supposed to work with the ev3 running pybricks?

The compatibility menu at the top of https://docs.pybricks.com/en/latest/iodevices/xboxcontroller.html doesn't list the ev3 at all. 

When I try to connect the ev3 to the following controller https://www.amazon.com/Controller-Officially-Licensed-Xbox-Windows-X/dp/B0BC2748QQ/ it never discovers the controller. It does discover and connect to a PC, and the controller also connects to PC fine. 

I have tried searching but can't find definitive answers. YouTube has some videos showing people controlling an ev3 with an Xbox controller, but the videos don't say much on how it's done.

Thanks

**Comment by laurensvalk** [comment]
> Are Xbox compatible controllers supposed to work with the ev3 running pybricks?

The `XboxController` class is not available for EV3. You can find the EV3 MicroPython documentation [here](https://pybricks.com/ev3-micropython/).

Since Pybricks for EV3 runs on `ev3dev` which is Debian Linux, you may be able to pair other controllers and interface with it through the filesystem.

All recent official Xbox Controllers use Bluetooth Low Energy though, which the EV3 does not support.
